.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+
+name: Feature Request
+about: Suggest an idea for DarkflameServer
+title: "ENH:"
+labels: "Enhancement, Needs Triage"
+
+---
+
+#### Is your feature request related to a problem?
```

Comment:
Is there a reason that only this template doesn't use the fancy fields, but just uses raw markdown instead?

Reply:
No not really, will actually change that :)

Reply:
Updated to also make use of the yaml syntax.